.changeset/metal-squids-look.md

```diff
@@ -0,0 +1,8 @@
+---
+"@wso2is/console": patch
+"@wso2is/myaccount": patch
+"@wso2is/identity-apps-core": patch
```

Comment:
For i18n, we don't need to bump identity-apps-core, since i18n for JSP apps are handled separately.

cc: @JayaShakthi97

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/4878/commits/0768eeb420800043326bf507143c2484558315c0